solutions/apigee/dev/main.tf

```diff
@@ -0,0 +1,126 @@
+resource "google_compute_instance" "gce_virtual_machine" {
+
+  name           = var.gce_name
+  machine_type   = var.gce_machine_type
+  zone           = var.gcp_zone
+  project        = var.gcp_project_id
+  tags           = ["http-server"]
+  can_ip_forward = var.gce_can_ip_forward
+
+  boot_disk {
+    initialize_params {
+      image = var.gce_machine_image
+    }
+  }
+
+  network_interface {
+    network    = var.gce_machine_network == "default" ? var.gce_machine_network : null
+    subnetwork = var.gce_machine_network == "default" ? null : var.gce_machine_network
+
+    access_config {
+      // Ephemeral IP
+    }
+  }
+
+  # Add Key/Value pair e.g. SSH keys here
+  metadata = {
+    childProjectZone            = "us-east1-b"
```

Comment:
Is hardcoding of zone required?

Reply:
@mahakverma-tudip Please update the metadata object to use variables. This object needs to be set per use case, so should not include hardcoded properties.